core/src/test/java/org/apache/calcite/test/SqlToRelTestBase.java

```diff
@@ -796,6 +853,43 @@ public void assertConvertsTo(
       diffRepos.assertEquals("plan", plan, actual);
     }
 
+    public RexNode convertExprToRex(String expr) {
+      Objects.requireNonNull(expr, "expr");
+      final SqlNode sqlQuery;
+      try {
+        sqlQuery = parseExpression(expr);
+      } catch (RuntimeException | Error e) {
+        throw e;
+      } catch (Exception e) {
+        throw TestUtil.rethrow(e);
+      }
+      final RelDataTypeFactory typeFactory = getTypeFactory();
+      final Prepare.CatalogReader catalogReader =
```

Comment:
Can we reuse the logic for creating `SqlToRelConverter` ?

Reply:
The duplicate code build both validator and SqlToRelConverter . Is it a good idea to let this code return Pair.of(SqlToRelConveter,SqlValidator)